.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 2
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Do we need this file since we have Prettier?

Reply:
This file is just a nice to have because it plays nice with editors... if you feel strongly though I can remove it.

Reply:
Yeah, but since we have Prettier, users can just use their default editor settings and that's the nice thing about Prettier that we don't need to align the editor settings with the project anymore.

Reply:
Done